File: code/core/hello-android/HelloActivity.scala
package com.decaffeinatedandroid.scala.hello_android

import android.app.Activity
import android.os.Bundle
import android.widget.TextView

class HelloActivity extends Activity {
  override def onCreate(savedState: Bundle) {
    super.onCreate(savedState)
    setContentView(
      new TextView(this) {
        setText("Hello, Android!")
      }
    )
  }
}

File: code/core/hello-android/HelloActivity.java
package com.decaffeinatedandroid.scala.hello_android

import android.app.Activity
import android.os.Bundle
import android.widget.TextView

public class HelloActivity extends Activity {
  @Override
  public void onCreate(Bundle savedState) {
    super.onCreate(savedState);
    TextView textView = new TextView(this);
    textView.setText("Hello, Android!");
    setContentView(textView);
  }
}
